src/storage/txn/mod.rs

```diff
@@ -122,6 +122,11 @@ quick_error! {
                         lower_bound.as_ref().map(hex::encode_upper).unwrap_or_else(|| "(none)".to_owned()),
                         upper_bound.as_ref().map(hex::encode_upper).unwrap_or_else(|| "(none)".to_owned()))
         }
+        MaxTimestampNotSynced { region_id: u64, start_ts: TimeStamp } {
```

Comment:
I think we can add these 2 fields to the kvproto too.

Reply:
Maybe not so necessary. The client itself knows its `region_id` and `start_ts`.

Reply:
ok